bin/bundler-audit

```diff
@@ -4,10 +4,12 @@ $LOAD_PATH.unshift(File.expand_path(File.join(File.dirname(__FILE__), "../lib"))
 
 require 'cc/engine/bundler_audit'
 
-def parse_engine_config(raw)
-  raw ? JSON.parse(raw) : {}
+engine_config = {}
+
+if File.exists?("/config.json")
+  engine_config = JSON.parse(File.read("/config.json"))
 end
```

Comment:
I've seen this pattern in a few places, I think we should decide on a style:

```rb
x = default

if condition
  x = something
end
```

Vs

```rb
if condition
  x = something
else
  x = default
end
```

I personally prefer the second, but I'm OK if everyone else prefers what you're doing.

No need to discuss this now, just wanted to raise it when an example came by.